charts/studio/templates/studio/_env.tpl

```diff
@@ -44,21 +44,20 @@ Keycloak URL
 {{- end -}}
 {{- end -}}
 
-{{/*
```

Comment:
The environment variables here were duplicated

Reply:
Doesn't backend need these variables?

Reply:
Hmm, then something isn't right, because they are for sure not there: https://github.com/RasaHQ/rasa-helm-charts/blob/main/charts/studio/values.yaml#L855

Maybe you kept all changes during rebase?

Reply:
Seems like the rebase was fine but we still haven't updated the studio deployment to use the `2.1.9` helm chart version so that is probably the problem 😱 I'll take a look in there to see if I can fix the duplication in my PR